evm_arithmetization/src/fixed_recursive_verifier.rs

```diff
@@ -238,9 +242,21 @@ impl<const D: usize> AggregationChildTarget<D> {
         builder: &mut CircuitBuilder<F, D>,
     ) -> PublicValuesTarget {
         let agg_pv = PublicValuesTarget::from_public_inputs(&self.agg_proof.public_inputs);
-        let evm_pv = PublicValuesTarget::from_public_inputs(&self.evm_proof.public_inputs);
+        let evm_pv = PublicValuesTarget::from_public_inputs(&self.base_proof.public_inputs);
         PublicValuesTarget::select(builder, self.is_agg, agg_pv, evm_pv)
     }
+
+    fn public_values_vec<F: RichField + Extendable<D>>(
+        &self,
+        builder: &mut CircuitBuilder<F, D>,
+    ) -> Vec<Target> {
+        zip_eq(
+            &self.agg_proof.public_inputs,
+            &self.base_proof.public_inputs,
+        )
+        .map(|(&agg_pv, &block_pv)| builder.select(self.is_agg, agg_pv, block_pv))
+        .collect()
```

Comment:
These are not public values but the entire public inputs (containing the public values, the VK, etc..)

Reply:
You are right.

Reply:
Renamed to `public_inputs`.